webapp/layouts/default.vue

```diff
@@ -192,7 +192,9 @@ export default {
       return this.$route.path.indexOf(url) === 0
     },
     unfolded: function() {
-      document.getElementById('nav-search-box').classList.add('unfolded')
+      if (window.innerWidth < 765) {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-09T20:55:32Z" title="Tuesday, July 9th 2019, 10:55:32 pm +02:00">Jul 9, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -192,7 +192,9 @@ export default {
       return this.$route.path.indexOf(url) === 0
     },
     unfolded: function() {
-      document.getElementById('nav-search-box').classList.add('unfolded')
+      if (window.innerWidth < 765) {
```


Please don't use globals like `window` or `document`. This will likely break in server side rendering. On the server, `window` is `undefined` and this line will very likely throw a null pointer error.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-07-10T04:55:42Z" title="Wednesday, July 10th 2019, 6:55:42 am +02:00">Jul 10, 2019</time>_

---


ok